Dockerfile

```diff
@@ -7,8 +7,12 @@ RUN \
 COPY \
     out/$VERSION/headless-shell/headless-shell \
     out/$VERSION/headless-shell/.stamp \
-    out/$VERSION/headless-shell/*.so \
```

Comment:
Why bother to list all the files?

1. `.so.1` and `.json` do not match `*.so`.
2. it's better to be explicit.